README.md

```diff
@@ -12,3 +12,24 @@ This code was developed in and depends on Databricks.
 More technical documentation will be added in the future, but for now see [Architechture](docs/architecture.md) and [Features & Upgrades](docs/features_upgrades.md).
 
 Note that there are two other version of this model that are stored in `deprecated/` that are not being maintained. 
+
+## Install
+
+This repo is designed to be run on Databricks. But it is also designed to use
```

Comment:
Thanks for adding this to the readme! The instructions were super easy to follow. I'd never used these init scripts before and learned something new while reviewing

Reply:
yes! I was planning on doing a little PR on dohyo soon to switch over to the newest dmlutils and to use the init script rather than installing at the notebook level.